internal/envstest/server.go

```diff
@@ -336,3 +354,23 @@ func ServerAssetsPath() string {
 func LocalesPath() string {
 	return filepath.Join(project.Root(), "internal", "i18n", "locales")
 }
+
+func randomBytes(tb testing.TB, len int) []byte {
```

Comment:
Nice - I was going for some of this in: https://github.com/google/exposure-notifications-verification-server/pull/1412